### DISCUSSION TITLE
Can't seem to enable and start pufferpanel

### DISCUSSION THREAD

**Discussion by Saphhmen** [pufferpanel/pufferpanel#1307] [Q&A]
When i enable the panel it sends an error "Failed to enable unit, unit pufferpanel.service does not exist"

**Comment by LordRalex** [comment]
How did you install the panel?
What OS?

**Reply by Saphhmen** [reply]
Ubuntu
I used the installation tutorial from the docs

**Reply by LordRalex** [reply]
Please provide more information. What version of Ubuntu.
What is the architecture of your machine?
What steps did you followed explicitly?

**Reply by Saphhmen** [reply]
> Please provide more information. What version of Ubuntu.
> What is the architecture of your machine?
> What steps did you followed explicitly?

I dont exactly know whats my ubuntu version, ive rarely use linux
As i said i used the instructions from the docs, the normal version not the docker

**Reply by LordRalex** [reply]
`uname -a`
`cat /etc/lsb-release`
`ls -l /var/lib/pufferpanel`
`apt search pufferpanel`

**Reply by Saphhmen** [reply]
Linux d5bc751cd98f 6.5.0-1014-gcp #14~22.04.1-Ubuntu SMP Sat Feb 10 04:57:00 UTC 2024 x86_64 GNU/Linux

**Reply by Saphhmen** [reply]
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.2 LTS

**Reply by Saphhmen** [reply]
ls: cannot access '/var/lib/pufferpanel': No such file or directory

**Reply by LordRalex** [reply]
` apt search pufferpanel`
